### DISCUSSION TITLE
Libre Link Connect !

### DISCUSSION THREAD

**Discussion by sircsaba** [NightscoutFoundation/xDrip#3720] [Q&A]
Everything is working perfect on my Google VM with NS. I would like to get the data from libreview to send it directly to NS. I already have a system setup with this configuration and its great (Raspberry Pi with the NS on docker and using timoschlueter uploader. I tried the connect with the proper information but that didn't work. Anyone using this method can send me the instructions please ?

**Comment by Navid200** [comment]
I have read your post multiple times and I cannot figure out what you are describing and what you need.

This is how xDrip can upload to Nightscout:
https://navid200.github.io/xDrip/docs/Nightscout-Settings.html  

These are xDrip forums.
In addition to that, we also support a different scheme for running Nightscut on Google Cloud.

Ask what you need instead of explaining what works.  If I can help, I will.

**Reply by sircsaba** [reply]
I would like to retrieve the cgm data from libreview to my NS in Google VM without using XDrip but using something like the timoschlueter uploader.

Is this clear enough ?

**Reply by Navid200** [reply]
You mean you want to upload CGM data from an app other than xDrip to Nightscout?

**Reply by sircsaba** [reply]
Yes

**Reply by PhilTU** [reply]
> You mean you want to upload CGM data from an app other than xDrip to Nightscout?

pull data from Libreview (like Dexcom Clarity) and push it into Nightscout.

**Reply by Navid200** [reply]
@PhilTU 
That's not what he wants to do.  He does not want to use xDrip.  I already clarified how xDrip can upload to Nightscout.  And xDrip obviously can get readings from many sources.  

I asked for clarification and this is what I got:

> without using XDrip

Which of course begs the question why ask in xDrip forums.  But, anyway, I went silent because it seems like a pointless conversation.

**Reply by sircsaba** [reply]
Wow, I thought that we are all in this together to help each other out no matter what we are using. Sorry for stepping on your delicate toes because I'm not using xDrip ... I'm using your NS for Google VM which is very nicely put together and I'm just wanting to expand on it for myself without using xDrip which I fond isn't necessary in my case and maybe others. Don't help if you want, that's your right but I'M not here for a "pointless conversation" I'm here for a solution and if anyone wants to help, please do and those who don't just abstain !

**Reply by sircsaba** [reply]
By the way, it's not using "another app", it's using another way to get the data without using an app ...

**Reply by PhilTU** [reply]
> @PhilTU That's not what he wants to do. He does not want to use xDrip.

I disagree, he wants to move data from Libreview into Nightscout according to what was written. Without xDrip yes.

**Reply by sircsaba** [reply]
Exactly and without any app on the phone so its all done on the cloud.

**Reply by Navid200** [reply]
I cannot abstain!  I have to coordinate these forums, which are here because I asked for them to be opened.
We have limited resources.  These are xDrip forums.  

You asked for help and you are welcome to any you will get.  I never objected to you.
Hopefully, Phil can help you.

**Reply by PhilTU** [reply]
> Exactly and without any app on the phone so its all done on the cloud.

Is the "accept permissions" thing getting in your way... https://forums.unraid.net/topic/148184-support-nightscout-librelink-up-uploader-placeholder/

**Comment by sircsaba** [comment]
Yes. In theory there's the "connect" variables that can be set but they aren't working or I'm doing something wrong ... If someone has this already setup and give me the details, it would be appreciated. This avoids having a bunch of apps on a phone other than the apps for the cgm and loop. The rest of the information is done automatically in the cloud.
Like I previously mentioned, I have a Raspberry PI with NS installed and timoschlueter uploader (https://github.com/timoschlueter/nightscout-librelink-up) which works great which I would like to have something similar for me NS on Google VM.

**Reply by sircsaba** [reply]
No, I already reported this when it actually happened on another forum and found the solution : https://github.com/timoschlueter/nightscout-librelink-up/issues/162

I was hoping to integrate this but I'm running it successfully on docker on the Google VM so this is a solution for now. Thanks for helping its appreciated. The whole goal for this and I'm sure for a lot of us is to be as independent as possible with the least amount of components as possible. Since these systems are health critical, I like having redundancy so if one system is down, I can switch effortlessly to another. Like many of us, I have a backup Loop phone ready to jump into action if ever the Looper phone gets disabled. 

I like to thank everyone from the xDrip group for all the effort in putting this all together. Its very nice to have these solutions readily and easily available.
